src/features/FavoriteList/FakeDoor/FavoriteListOfferModal.tsx

```diff
@@ -20,21 +20,34 @@ export const FavoriteListOfferModal: React.FC<Props> = ({
   hideModal,
   showSurveyModal,
 }) => {
+  const [shouldSeeSurvey, setShouldSeeSurvey] = useState(false)
+  const onCreateFavListPress = () => {
+    setShouldSeeSurvey(true)
+    hideModal()
+  }
+  const onHideModalPress = () => {
+    setShouldSeeSurvey(false)
+    hideModal()
+  }
   return (
     <React.Fragment>
       <AppModalWithIllustration
         Illustration={StyledBicolorTables}
         title="Crée une liste de favoris&nbsp;!"
-        hideModal={hideModal}
-        onModalHide={showSurveyModal}
+        hideModal={onHideModalPress}
+        onModalHide={shouldSeeSurvey ? showSurveyModal : undefined}
```

Comment:
Je ne sais pas si c'est mieux ? :D
```suggestion
        onModalHide={shouldSeeSurvey && showSurveyModal}
```